### PR TITLE
Fixed pylint issues in five modules and deleted one unused file

### PR DESCRIPTION
**Pull request by asankov**

### CODE REVIEW

File: tcms/core/contrib/auth/views.py [hunk 1]
```diff
@@ -36,7 +35,7 @@ def register(request, template_name='registration/registration_form.html'):
         form = RegistrationForm(data=request.POST, files=request.FILES)
         if form.is_valid():
             new_user = form.save()
-            ak = form.set_active_key()
+            active_key = form.set_active_key()
```
Comment (atodorov) [inline tcms/core/contrib/auth/views.py]: This is activation_key and both the form method and the send_mail parameter bame needs to reflect that

File: tcms/core/utils/__init__.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 from django.conf import settings
 
-numeric_re = re.compile(r'^\d+$')
+NUMERIC_RE = re.compile(r'^\d+$')
 
 
-def is_int(s):
-    return numeric_re.match(s) is not None
+def is_int(string):
+    return NUMERIC_RE.match(string) is not None
```
Comment (atodorov) [inline tcms/core/utils/__init__.py]: Is this constant used elsewhere ?

Also is is_int used in many places ? Sounds like something that can be removed.

File: tcms/wsgi.py [hunk 3]
```diff
@@ -36,15 +36,15 @@
 # This application object is used by any WSGI server configured to use this
 # file. This includes Django's development server, if the WSGI_APPLICATION
 # setting points here.
-_application = get_wsgi_application()
+_APPLICATION = get_wsgi_application()
```
Comment (atodorov) [inline tcms/wsgi.py]: It looks like you can move the get_wsgi_application call down in the application() function

File: tcms/core/utils/__init__.py (hunk 2 above)
Comment (asankov) [reply tcms/core/utils/__init__.py]: `NUMERIC_RE` is not used anywhere else, and `is_int` is used just in one place. I may think of a way to remove them.

File: tcms/wsgi.py [hunk 4]
```diff
@@ -33,18 +33,13 @@
 # add tcms's core lib path
 sys.path.append(os.path.join(tcms.__path__[0], 'core', 'lib'))
 
-# This application object is used by any WSGI server configured to use this
-# file. This includes Django's development server, if the WSGI_APPLICATION
-# setting points here.
-_application = get_wsgi_application()
```
Comment (atodorov) [inline tcms/wsgi.py]: Drop this commit for now. After further review I think we should keep the _application variable as it seems to provide some sort of caching local to the module. get_wsgi_application() does setup a Django instance which may not be desirable every time. However I'm not sure how the application() function works with Django nowadays.

File: tcms/core/contrib/auth/views.py [hunk 5]
```diff
@@ -87,8 +86,8 @@ def confirm(request, activation_key):
 
     # Get the object
     try:
-        ak = UserActivateKey.objects.select_related('user')
-        ak = ak.get(activation_key=activation_key)
+        _activation_key = UserActivateKey.objects.select_related('user')
+        _activation_key = _activation_key.get(activation_key=activation_key)
```
Comment (atodorov) [inline tcms/core/contrib/auth/views.py]: Tip: next time you can squash these 2 lines together.

**Pull request merged**